pybotx/client/stickers_api/create_sticker_pack.py

```diff
@@ -2,16 +2,22 @@
 from uuid import UUID
 
 from pybotx.client.authorized_botx_method import AuthorizedBotXMethod
+from pybotx.missing import Missing
 from pybotx.models.api_base import UnverifiedPayloadBaseModel, VerifiedPayloadBaseModel
 from pybotx.models.stickers import StickerPack
 
 
 class BotXAPICreateStickerPackRequestPayload(UnverifiedPayloadBaseModel):
     name: str
+    user_huid: UUID
```

Comment:
`user_huid` here must also be `Missing[UUID]`

Reply:
Done